### DISCUSSION TITLE
quick question -- what is the key board in this photo?

### DISCUSSION THREAD

**Discussion by diflouroethane** [shorepine/tulipcc#312] [General]
![image](https://github.com/user-attachments/assets/ae228c32-e0e1-4103-b6e6-0a7d129a2236)
or if there are keyboards in similar sizes.

**Comment by bwhitman** [comment]
That's a 40% ortholinear keyboard. I got it from here https://kprepublic.com/products/bm40-rgb-40-hot-swap-custom-mechanical-keyboard-pcb-qmk-underglow-type-c-planck

**Reply by diflouroethane** [reply]
thanks!
